src/main/java/com/snowflake/kafka/connector/internal/streaming/BufferedTopicPartitionChannel.java

```diff
@@ -669,6 +669,10 @@ public InsertRowsResponse get() throws Throwable {
               // Simply added to the final response if it's not schema related errors
               finalResponse.addError(insertError);
             } else {
+              LOGGER.info(
+                  "Triggering schema evolution. NonNullableColumns={}, extraColumns={}",
+                  String.join("", nonNullableColumns),
```

Comment:
Maybe we want a spacebar or other separator for `nonNullableColumns`? (I didnt check how it looks now)

Reply:
Good catch I missed that.